raiden/tests/integration/test_raidenservice.py

```diff
@@ -204,130 +184,3 @@ def first_run_with_check(last_log_block):
     # If all runs well and our first_run_with_check function runs then test passes
     # since that means channels were queriable right after the first run of the
     # alarm task
-
-
-@pytest.mark.parametrize("number_of_nodes", [2])
-def test_fees_are_updated_during_startup(
```

Comment:
Is there any reason that the node shouldn't send it's fees on startup?

Reply:
Same as above.